apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/PooledSession.cs

```diff
@@ -58,6 +58,24 @@ public sealed class PooledSession : IDisposable
         /// </summary>
         internal ModeOneofCase TransactionMode { get; }
 
+        internal TransactionOptions TransactionOptions
```

Comment:
Optional nit: might want to make this a method, given that it creates a new instance each time.